### PR TITLE
Pylint

### PR DESCRIPTION
**Pull request by 1Revenger1**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ jobs:
       - name: Run tests
         working-directory: ./camera/auth
         run: make test
+      - name: Run lint
+        working-directory: ./camera/auth
+        run: make lint
```
Comment (tucker-moore) [inline .github/workflows/main.yml:25]: Should we run the linter before we run any tests? It looks like tests take ~1min, we would get quicker feedback if we ran the linter before our tests

**Review by tucker-moore — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (1Revenger1) [reply .github/workflows/main.yml:25]: Vscode shows the linter results already, I think it's fine to leave as is. I'm not sure how important quick feedback is here.

Comment (1Revenger1) [reply .github/workflows/main.yml:25]: Tests passing is also a higher priority compared to linting. Github actions stop executing commands on a failure iirc.

**Review by tucker-moore — APPROVED** [review]

File: camera/auth/src/ble/dbus_interface/dbus_bluez_interface.py [hunk 2]
```diff
@@ -53,94 +65,103 @@ def __init__(self, bus, uuid, index, primary):
...
             raise InvalidArgsException()
         return self.props
 
 class Characteristic(dbus.service.Object):
-    def __init__(self, bus, uuid, flags, service, index):
-        self.path = service.get_path() + "/char" + str(index)
-        self.descriptors = []
+    """BlueZ Characteristic which acts as an endpoint another BLE devices can access"""
+
+    def __init__(self, service, **kwargs):
```
[the remark this answers is not in the document]
Comment (1Revenger1) [reply camera/auth/src/ble/dbus_interface/dbus_bluez_interface.py:94]: Probably? I doubt we'll be using new characteristics though and it's not that hard to look at the other characteristics.

[the remark this answers is not in the document]
Comment (1Revenger1) [reply camera/auth/src/ble/dbus_interface/dbus_bluez_interface.py:94]: If this was a longer term project I'd probably say yes :P

**Pull request merged**